types/setting.go

```diff
@@ -1052,14 +1055,30 @@ var (
 	SettingDefinitionOrphanAutoDeletion = SettingDefinition{
 		DisplayName: "Orphan Auto-Deletion",
 		Description: "This setting allows Longhorn to delete the orphan resource and its corresponding orphaned data automatically. \n\n" +
-			"Orphan resources on down or unknown nodes will not be cleaned up automatically. \n\n",
+			"Orphan resources on down or unknown nodes will not be cleaned up automatically. \n\n" +
+			fmt.Sprintf("Deprecated: enable \"%s\" in %s instead. \n\n", OrphanResourceTypeReplicaData, SettingNameOrphanResourceAutoDeletion),
 		Category: SettingCategoryOrphan,
 		Type:     SettingTypeBool,
 		Required: true,
-		ReadOnly: false,
+		ReadOnly: true,
 		Default:  "false",
 	}
 
+	SettingDefinitionOrphanResourceAutoDeletion = SettingDefinition{
+		DisplayName: "Orphan Resource Auto-Deletion",
+		Description: "This setting allows Longhorn to delete the orphan resource and its corresponding orphaned resource automatically. \n\n" +
+			"Orphan resources on down or unknown nodes will not be cleaned up automatically. \n\n" +
+			"List the enabled resource types in a semicolon-separate list. Available items are: \n\n" +
+			"  - **replicaData**: replica data store \n\n" +
+			"  - **engineInstance**: engine runtime instance \n\n" +
+			"  - **replicaInstance**: replica runtime instance \n\n",
```

Comment:
This is user-facing content. Consider aligning with other setting descriptions for consistency.
```suggestion
			"List the enabled resource types in a semicolon-separate list. \n\n" +
			"Available items are: \n\n" +
			"- **replicaData**: replica data store \n\n" +
			"- **engineInstance**: engine runtime instance \n\n" +
			"- **replicaInstance**: replica runtime instance \n\n",
```